test/release/examples/primers/classes.chpl

```diff
@@ -22,7 +22,12 @@ class C {
 // each field in the class.  Once a class has been constructed, its methods
 // can be called.
 //
-var foo = new unmanaged C(1, 2);
+// Class types include a strategy for freeing them. Below it is ``owned``
+// but we'll discuss these options more below.
```

Comment:
I would say either "we'll discuss these options more later" or "Here it is ``owned``", it feels a little confusing otherwise